Lib/glyphsLib/__init__.py

```diff
@@ -54,6 +54,13 @@ def loads(value):
     return data
 
 
+def normalize(value):
+    """Read a .glyphs file from a bytes object.
```

Comment:
why bytes? It's better to work with strings (`unicode`, or `str` on py3) since this is really text, not binary data.
The parser should be able to handle both bytes or unicode input: the former has to be decoded from utf-8, the latter used as is.
Similarly the writer should work with unicode strings internally and then encode them to utf-8 when writing out to a "binary" file/stream, or write unicode directly to a file opened for "text" (which handles the encoding for us), or to an `io.StringIO` which is a stream of unicode strings.  

https://github.com/googlei18n/glyphsLib/pull/213